frontend/src/metabase/modes/components/drill/SortAction.jsx

```diff
@@ -1,5 +1,9 @@
 import { t } from "ttag";
 import Dimension from "metabase-lib/lib/Dimension";
+import _ from "underscore";
+import { TYPE, isa } from "metabase/lib/types";
+
+const INVALID_TYPES = [TYPE.Structured];
```

Comment:
This is being declared over and over again. Should we consider moving this somewhere like `constants.ts`, although I'm not sure what directory it should reside? And maybe name it something along the line `UNFILTERABLE_TYPES` so we know how they're not valid.

Reply:
I think we should leave them separate in each file because we've potentially got a few different things: some field types are unsortable, some are unfilterable, some should not be summarized, etc. It just so happens that you can't do anything on structured types yet. I suspect we may implement filtering before sorting or summarizing, so we should keep them separate.

Reply:
Good point